src/main/java/com/riwi/localstorage/riwi_local_storage/infrastructure/mappers/InventaryCreateMapper.java

```diff
@@ -0,0 +1,33 @@
+package com.riwi.localstorage.riwi_local_storage.infrastructure.mappers;
+
+import java.util.List;
+
+import org.mapstruct.InheritConfiguration;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingConstants;
+import org.mapstruct.Mappings;
+
+import com.riwi.localstorage.riwi_local_storage.api.dto.request.create.InventaryRequest;
+import com.riwi.localstorage.riwi_local_storage.api.dto.response.InventaryResponse;
+import com.riwi.localstorage.riwi_local_storage.domain.entities.Inventory;
+
+@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)
+public interface InventaryCreateMapper {
+    @Mappings({

```

Comment:
delete this 
    @Mappings({
        @Mapping(target = "id",ignore = true),
        @Mapping(target = "saleDetails",ignore = true),
        @Mapping(target = "supplierOrders",ignore = true),
        @Mapping(target = "product",ignore = true),
        @Mapping(target = "branch",ignore = true)

    })